ma_search/__init__.py

```diff
@@ -17,5 +17,65 @@
 limitations under the License.
 """
 
+import os
+import sys
+import logging
+
+from ma_search.config import Config
+
 __package__ = "ma_search"
 __version__ = "0.0.1"
+
+def _initLogging(logObj):
+    """Call to initialise logging
+    """
+    # Read environment variables
+    wantLevel = os.environ.get("MA_SEARCH_LOGLEVEL", "INFO")
+    logFile = os.environ.get("MA_SEARCH_LOGFILE", None)
+
+    # Determine log level and format
+    if hasattr(logging, wantLevel):
+        logLevel = getattr(logging, wantLevel)
+    else:
+        print("Invalid logging level '%s' in environment variable MA_SEARCH_LOGLEVEL" % wantLevel)
```

Comment:
shouldn't this be either a warning, or log.warning/log.error?
[not print, which is a sideeffect]

Reply:
This is before logging is fully initialised, so no, it should be print to make sure it is properly returned.

Reply:
still. you may use warnings, which can be filtered away,
```
import warnings
warnings.warn(...)
```

Reply:
This is an error during initialisation and should absolutely not be filtered away. I don't think this is the right approach. I think in fact this should cause a sys.exit instead.

Reply:
`raise ValueError`?

Reply:
Maybe. Just need to check what the parent process needs of return codes.